docs/CHANGELOG.asciidoc

```diff
@@ -51,6 +52,222 @@ Review important information about the {kib} 8.x releases.
+Platform::
+* ES|QL, a new query language, is available under technical preview in Discover and Dashboards ({kibana-pull}146971[#146971]).
+Querying & Filtering::
+* Saved queries can now be shared between multiple spaces ({kibana-pull}163436[#163436]).
+Uptime::
+* Discover document viewer has been added to the summary table ({kibana-pull}163926[#163926]).
+
+For more information about the features introduced in 8.11.0, refer to <<whats-new,What's new in 8.11>>.
+
+[[enhancements-and-bug-fixes-v8.11.0]]
+=== Enhancements and bug fixes
+
+For detailed information about the 8.11.0 release, review the enhancements and bug fixes.
+
+[float]
+[[enhancement-v8.11.0]]
+=== Enhancements
+APM::
+
+* Changed mobile badge from 'technical preview' to 'beta' ({kibana-pull}167543[#167543]).
+* New Profiling ES Flamegraph API ({kibana-pull}167477[#167477]).
+* Adds Universal Profiling to O11y overview and Setup guide ({kibana-pull}165092[#165092]).
+* Mark disabled alerts as Untracked in both Stack Management and o11y ({kibana-pull}164788[#164788]).
+* Adds time range to event metadata API ({kibana-pull}167132[#167132]).
+* New settings to control CO2 calculation ({kibana-pull}166637[#166637]).
+* Adds permissions for "input-only" package ({kibana-pull}166234[#166234]).
+* Adds selecting the consumer based on the authorized consumers when a user is creating an ES Query threshold rule ({kibana-pull}166032[#166032]).
+* Migrate Ace based `EuiCodeEditor` to Monaco based code editor ({kibana-pull}165951[#165951]).
+* Mobile UI crash widget added ({kibana-pull}163527[#163527]).
+Cases::
+* Show a warning message to inform user that navigating after the 10Kth case is not possible ({kibana-pull}164323[#164323]).
+Dashboard::
+* Focus on a single panel while disabling all other panels ({kibana-pull}165417[#165417]).
+* Adds filter details to panel settings ({kibana-pull}162913[#162913]).
+Discover::
+* Redesign for the grid, panels and sidebar ({kibana-pull}165866[#165866]).
+* Set data table row height to auto-fit by default ({kibana-pull}164218[#164218]).
+* Allow fetching more documents on Discover page ({kibana-pull}163784[#163784]).
+* Fixes unified search for long field inputs ({kibana-pull}166024[#166024]).
+Elastic Security::
+For the Elastic Security 8.11.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+Enterprise Search::
+For the Elastic Enterprise Search 8.11.0 release information, refer to {enterprise-search-ref}/changelog.html[_Elastic Enterprise Search Documentation Release notes_].
+Fleet::
+* Adds sidebar navigation showing headings extracted from the readme ({kibana-pull}167216[#167216]).
+Lens & Visualizations::
+* Other bucket defaults to false for top values greater than equal 1000 in *Lens* ({kibana-pull}167141[#167141]).
+* Adds support for decimals in percentiles in *Lens* ({kibana-pull}165703[#165703]).
+* Support date fields in options list ({kibana-pull}164362[#164362]).
```

Comment:
Are we adding the controls enhancements in the Lens section? I think we should move it to dashboard section and specify that this is about controls.